cmd/publicize/main.go

```diff
@@ -0,0 +1,189 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/ghodss/yaml"
+	"github.com/sirupsen/logrus"
+
+	kerrors "k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/apimachinery/pkg/util/sets"
+
+	"k8s.io/test-infra/prow/githubeventserver"
+	"k8s.io/test-infra/prow/interrupts"
+	"k8s.io/test-infra/prow/logrusutil"
+
+	"k8s.io/test-infra/pkg/flagutil"
+	"k8s.io/test-infra/prow/config/secret"
+	prowflagutil "k8s.io/test-infra/prow/flagutil"
+)
+
+type Config struct {
+	Bonds []Bond `json:"bonds,omitempty"`
+}
```

Comment:
Is there a reason why this is struct rather than `type Config Bonds[]Bond`. Extensibility?

Does it make sense to have empty slice?

Reply:
This is needed to allow Config to filled from a YAML file.